tests/vmi_networking_test.go

```diff
@@ -895,6 +895,13 @@ var _ = Describe("[Serial][rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][le
 
 					return nil
 				}, 120*time.Second).Should(Succeed())
+
+				By("Restarting the vmi")
+				Expect(console.ExpectBatch(vmi, []expect.Batcher{
+					&expect.BSnd{S: "sudo shutdown -r now\n"},
+				}, 10*time.Second)).To(Succeed())
+				tests.WaitUntilVMIReady(vmi, libnet.WithIPv6(loginMethod))
```

Comment:
`configureIpv6` was used before (line 871). Leaving it like this will always configure IPv6 even when the test is ipv4-only.

Reply:
done